Sources/BraintreeCore/BTWebAuthenticationSession.swift

```diff
@@ -1,12 +1,12 @@
 import Foundation
 import AuthenticationServices
 
-#if canImport(BraintreeCore)
-import BraintreeCore
-#endif
+///  :nodoc: This method is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
+@_documentation(visibility: private)
+public class BTWebAuthenticationSession: NSObject {
```

Comment:
Since we also use this in PPCP, do we think it would be worth it to ship this as it's own module to share between both SDKs?

Reply:
Bundling that ^ into a package with our ASWebWrapper might also make more of a case for it to be it's own module.

Reply:
I am certainly down to de-duplicate the logic further within the SDK.

re moving things into their own package, where are we thinking these packages would live? The Braintree namespace on GitHub or PayPal? I am down to move them but assumed we were a long ways off from standing up a new project/repo for these shared modules, publishing them for each package manager, pulling them in for each package manager, etc. But maybe that's all already been decided from the product side?

Reply:
> Yeh. We also have[ this logic](https://github.com/braintree/braintree_ios/blob/66a37b5ea72e0440b50ec59c869f12c5392dab63/Sources/BraintreePayPal/BTPayPalClient.swift#L445-L459) duplicated in a few places. Another punt, but we should mess around with ways to consolidate it.

Agree with this - it was already on my list to talk about with you if we want to pair later. 😄

Reply:
Moved the duplicated logic into it's own class here: https://github.com/braintree/braintree_ios/pull/1044/commits/0780eab9e7f265ac451eebee142799f93bcc6214. Going to 🏈  on moving it into it's own module. These classes are all only public for SDK use so we can do this in a minor version at any time.